Marlin/src/HAL/TEENSY35_36/HAL_SPI.cpp

```diff
@@ -42,18 +42,11 @@ void spiBegin() {
   SET_OUTPUT(SD_SCK_PIN);
   SET_INPUT(SD_MISO_PIN);
   SET_OUTPUT(SD_MOSI_PIN);
-
-  #if 0 && DISABLED(SOFTWARE_SPI)
-    // set SS high - may be chip select for another SPI device
-    #if SET_SPI_SS_HIGH
```

Comment:
Since `SET_SPI_SS_HIGH` still exists we should look at whether it is still relevant for startup.

Reply:
The only other mention is at SdFatConfig.h line 79 where it is being defined. The removed mentions above were disabled, see the "#if 0", thus it can be considered a deprecated thing. What is your argument for investigating the chip-select?

Reply:
I'm wondering if there are some devices in circulation that still require the behavior described with `SET_SPI_SS_HIGH`, and whether we should implement that if it is currently missing, and perhaps use that option if it ends up part of the SD startup within the SDFat files.
```cpp
/**
 * Set the SS pin high for hardware SPI.  If SS is chip select for another SPI
 * device this will disable that device during the SD init phase.
 */
```

Reply:
I wonder, too! I cannot tell without consulting any documentation. If there is any further hint that does not require deep investigation then please tell us.